### PR TITLE
Cleanup from `webform` support

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

**Review by mal-tee — APPROVED** [review]
Thank you!

File: src/Components/RequestList.tsx [hunk 1]
```diff
@@ -392,7 +397,13 @@ export const ProceedingRow = (props: ProceedingRowProps) => {
                                     msg.subject && (
                                         <>
                                             <br />
-                                            <span className="icon icon-email">{msg.subject}</span>
+                                            <span
+                                                className={`icon ${iconClassForTransportMedium[msg.transport_medium]}`}
+                                                title={t('sent-via-medium', 'my-requests', {
+                                                    medium: t(msg.transport_medium, 'my-requests'),
+                                                })}>
```
Comment (zner0L) [inline src/Components/RequestList.tsx:404]: This should actually also be an `aria-label` (https://www.a11yproject.com/posts/title-attributes/).

File: src/Components/RequestList.tsx [hunk 2]
```diff
@@ -358,7 +358,12 @@ export const ProceedingRow = (props: ProceedingRowProps) => {
...
                                         <br />
                                         <button
-                                            className="button button-unstyled icon icon-email"
+                                            className={`button button-unstyled icon ${
+                                                iconClassForTransportMedium[msg.transport_medium]
+                                            }`}
+                                            title={t('sent-via-medium', 'my-requests', {
+                                                medium: t(msg.transport_medium, 'my-requests'),
+                                            })}
                                             onClick={() => {
```
Comment (zner0L) [inline src/Components/RequestList.tsx:367]: Same here. This needs an `aria-label` as well.

Comment (zner0L) [reply src/Components/RequestList.tsx:367]: I guess an eslint rule would be useful for that.

**Review by zner0L — CHANGES_REQUESTED** [review]
Looks good, just a small a11y issue.

**Review by zner0L — APPROVED** [review]

**Pull request merged**